h2o-docs/src/product/grid-search.rst

```diff
@@ -85,6 +89,169 @@ Grid Search in R and Python
         {'strategy': "RandomDiscrete", 'stopping_tolerance': 0.001, 'stopping_rounds': 10}
         {'strategy': "RandomDiscrete", 'stopping_metric': "misclassification", 'stopping_tolerance': 0.0005, 'stopping_rounds': 5}
 
+Saving and Loading a Grid Search
```

Comment:
These examples should go after the Grid Search and Random Grid Search examples. (Basically, you'd want to describe how to build grid search models before describing how to save them.) Also, maybe these can be a conclusion of the Grid Search example rather than a completely new one.

Reply:
The examples now use the Grid Search example--please let me know if how I integrated it makes sense!